pkg/codegen/go/gen.go

```diff
@@ -2970,8 +2971,11 @@ func (pkg *pkgContext) genConfig(w io.Writer, variables []*schema.Property) erro
 		"github.com/pulumi/pulumi/sdk/v3/go/pulumi":        "",
 	}
 	pkg.getImports(variables, importsAndAliases)
+	importsAndAliases[path.Join(pkg.importBasePath, "internal")] = ""
+	pkg.genHeader(w, nil, importsAndAliases, notUtil)
 
-	pkg.genHeader(w, nil, importsAndAliases)
+	// in case we're not using the internal package, assign to a blank var
```

Comment:
Can't we do the old "needsUtil" trick to decide if we need to import it or not?

Reply:
Yeh it's minor but it would save needing to emit this blank var in every file.

Reply:
update: because we generate the import headers _before_ determining if we need to use the internal package logic, we would have to rewrite how we generate import headers, which is more risky.
Speaking offline, we determined that this is a non-urgent nice-to-have.